pkgs/development/tools/mblock-mlink/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchurl, dpkg, makeWrapper, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "mblock-mlink";
+  version = "1.2.0";
+
+  src = fetchurl {
+    url = "https://dl.makeblock.com/mblock5/linux/mLink-${version}-amd64.deb";
+    sha256 = "sha256-KLxj81ZjbEvhhaz0seNB4WXX5ybeZ7/WcT1dGfdWle0=";
+  };
+
+  unpackPhase = ''
+    ${dpkg}/bin/dpkg -x $src $out
+  '';
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    echo $out/usr/local/makeblock/mLink/mnode $out/usr/local/makeblock/mLink/app.js > $out/bin/mlink
```

```suggestion
    echo $out/usr/makeblock/mLink/mnode $out/usr/makeblock/mLink/app.js > $out/bin/mlink
```
and please move the files. local is reserved for manually installed things outside of your package manager.

---

Hi, that's just how it comes in the deb :man_shrugging: 
I guess I could move things after unpacking if you really want?

---

Done in 84fe9e42ff55f9905e27df62ee12ecb1c843267d